src/scripts/destinyTrackerApi/itemListBuilder.js

```diff
@@ -0,0 +1,74 @@
+import _ from 'underscore';
+import { itemTransformer } from './itemTransformer.js';
+
+class itemListBuilder {
```

Comment:
Side note - the more I work with them, the more convinced I am that ES2015 classes were a bad design... I prefer the control you have when building a normal object and exposing only the functionality you intend to be part of a public API...